android/src/main/java/com/polidea/blemulator/SimulatorPlugin.java

```diff
@@ -17,6 +18,7 @@
 public class SimulatorPlugin implements MethodChannel.MethodCallHandler {
 
     private DartMethodCaller dartMethodCaller;
+    private DartValueHandler dartValueHandler;
 
     public static void registerWith(PluginRegistry.Registrar registrar) {
         MethodChannel downstream = new MethodChannel(registrar.messenger(), ChannelName.TO_JAVA);
```

Comment:
Just a suggestion. Maybe we should get rid of all downstream/upstream conventions since we partially did so?